core/modules/user/user.pages.inc

```diff
@@ -485,29 +487,53 @@ function user_cancel_confirm_form_submit($form, &$form_state) {
+
   $methods = array(
     'user_cancel_block' => array(
       'title' => t('Disable the account and keep its content.'),
-      'description' => t('Your account will be blocked and you will no longer be able to log in. All of your content will remain attributed to your user name.'),
+      'description' => theme('item_list', array('items' => array(
+          t('Your account will be blocked, and you will no longer be able to log in.'),
+          t('All of your content will remain attributed to your user name.'),
+        )
+      )),
     ),
     'user_cancel_block_unpublish' => array(
       'title' => t('Disable the account and unpublish its content.'),
-      'description' => t('Your account will be blocked and you will no longer be able to log in. All of your content will be hidden from everyone but administrators.'),
+      'description' => theme('item_list', array('items' => array(
+          t('Your account will be blocked, and you will no longer be able to log in.'),
+          t('All of your content will be hidden from everyone but administrators.'),
+        )
+      )),
     ),
     'user_cancel_reassign' => array(
-      'title' => t('Delete the account and make its content belong to the %anonymous-name user.', array('%anonymous-name' => $anonymous_name)),
-      'description' => t('Your account will be removed and all account information deleted. All of your content will be assigned to the %anonymous-name user.', array('%anonymous-name' => $anonymous_name)),
+      'title' => t('Permanently delete the account, and make its content belong to the %anonymous-name user.', array('%anonymous-name' => $anonymous_name)),
+      'description' => theme('item_list', array('items' => array(
+          t('Your account will be <u><em>permanently</em></u> removed, and all account information deleted.'),
+          t('All of your content will be assigned to the %anonymous-name user.', array('%anonymous-name' => $anonymous_name)),
+        )
+      )),
+      'danger' => TRUE,
     ),
     'user_cancel_delete' => array(
-      'title' => t('Delete the account and its content.'),
-      'description' => t('Your account will be removed and all account information deleted. All of your content will also be deleted.'),
+      'title' => t('Permanently delete the account and its content.'),
+      'description' => theme('item_list', array('items' => array(
+          t('Your account will be <u><em>permanently</em></u> removed, and all account information deleted.'),
+          t('All of your content will also be deleted.'),
+        )
+      )),
+      'danger' => TRUE,
+      // This was meant to be used in user_admin_settings() in order to remove
+      // cancellation methods that have #access defined, as those cannot be
+      // configured as the default method. However this property was never being
+      // returned by this function here before. ¯\_(ツ)_/¯
       'access' => user_access('administer users'),
```

Comment:
This pre-exiting code was meant to be used in user_admin_settings() in order to remove cancellation methods that have `#access` defined, as those cannot be configured as the default method. However this property was never being returned by this function here before 🤷🏼 

I've raised https://github.com/backdrop/backdrop-issues/issues/5121 for that.